public/app/plugins/datasource/azuremonitor/dashboards/containerInsightsSyslog.json

```diff
@@ -0,0 +1,1291 @@
+{
+    "__inputs": [],
+    "__elements": {},
+    "__requires": [
+      {
```

Comment:
This panel won't be available in Grafana generally and so should be removed.

Reply:
I am a bit confused here on which lines I should be removing. Also, I have run prettier on the file on my end and am not seeing any updates to the formatting, is there something else I should be doing?

Reply:
R.E. prettier are you using the command available in Grafana? `yarn prettier:write`

Reply:
Believe I have corrected the formatting issues. Please let me know if there's anything else you need me to do before we can get this merged!